src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -334,6 +334,10 @@
                 <source>This value should be valid JSON.</source>
                 <target>Cette valeur doit être un JSON valide.</target>
             </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Cette collection ne doit contenir que des éléments uniques.</target>
```

Comment:
des éléments différents ?

Reply:
Changed ;)

Reply:
Ou maybe "Cette collection ne doit pas comporter de doublons", or do we have to not wander too much away when translating?

Reply:
Even better!